packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -9,20 +9,20 @@ const GlobalClassNames = {
 };
 
 export const getStyles = (props: ICheckStyleProps): ICheckStyles => {
-  const { checkBoxHeight = '18px', checked, className, theme } = props;
+  const { height = props.checkBoxHeight || '18px', checked, className, theme } = props;
```

Comment:
Could this just be `checkBoxHeight: height = '18px'`?

Reply:
No, `height` is the new prop for height, and I want to make it default to the value of the old prop for height (`checkBoxHeight`).

Reply:
This is more of a general cleanup change, not core to the PR--the original `checkBoxHeight` prop name seemed unnecessarily long/redundant.